services/frontend/nginx.conf.template

```diff
@@ -49,9 +49,12 @@ http {
 
     location / {
       try_files ${ESC}uri /index.html;
-      # Short lived cache for non-asset pages
-      expires 5m;
-      add_header Cache-Control "public";
+      # From https://quasar.dev/quasar-cli-vite/developing-spa/deploying/
+      #
+      # > It’s important that you do not allow browsers to cache the index.html file.
+      # > Because otherwise updates to this file or to your app might slip through the cracks
+      # > for browsers that load the index.html from cache.
+      add_header Cache-Control "no-cache, no-store, must-revalidate";
```

Comment:
It's a bummer to lose index caching, but it was already not configured for very aggressive caching.

Reply:
This seems like it'll be pretty bulletproof. Do we really need the other solution if we merge this? Or is the PR just two independent solutions for review then we pick one?

Reply:
Well, originally I was thinking of it as a belt and suspenders thing.

The weird thing is that the previous behavior (unless I messed it up) should have only cached the index for like 5 minutes. But that doesn't mesh with the reports of people getting the cached page well after the deploy, and for extended periods of time.

So I guess I don't trust my ability to have this cache rule fully respected.